tests/posix/env/prj.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_ZTEST=y
 CONFIG_POSIX_ENV=y
+CONFIG_PICOLIBC=y
```

Comment:
It would be helpful to leave a comment above this option just to clarify why this was added.

The reason is that it's a bit unusual to see defaults explicitly set in `prj.conf` (convention over configuration).

Reply:
Done